src/cascadia/TerminalApp/App.cpp

```diff
@@ -222,8 +219,8 @@ namespace winrt::TerminalApp::implementation
                                     const winrt::hstring& contentKey,
                                     HRESULT settingsLoadedResult)
     {
-        auto title = _resourceLoader->GetLocalizedString(titleKey);
-        auto buttonText = _resourceLoader->GetLocalizedString(L"Ok");
+        auto title = GetLibraryResourceString(titleKey);

```

Comment:
Nah, this can't be `RS_` -- RS will try to emit a static resource reference into that binary segment, but it can't because this is not a static constant value.

Reply:
i think this will fail at compile time, but in an unclear way